docs/calamari/SDK.md

```diff
@@ -0,0 +1,292 @@
+# 🛠 Calamari SDK
+
+This package implements a Javascript SDK for connecting with the Manta Network. API end points to connect to Calamari:
+
+## Full Nodes
+
+- provided by Manta: `wss://ws.calamari.systems`
+- provided by third party: `wss://calamari.api.onfinality.io`
+
+## Asset API (Javascript)
+
+- KMA: `api.query.system.account(AccountId)`
+- KAR: `api.query.assets.account(8u128, AccountId)`
+- AUSD: `api.query.assets.account(9u128, AccountId)`
+- LKSM: `api.query.assets.account(10u128, AccountId)`
+- MOVR: `api.query.assets.account(11u128, AccountId)`
+- KSM: `api.query.assets.account(12u128, AccountId)`
+- PHA: `api.query.assets.account(13u128, AccountId)`
+
+## Installation
+
+```sh
+yarn install manta.js
+```
+
+### Local Development
+
+1. `git clone https://github.com/Manta-Network/sdk.git`
+2. `cd sdk/manta-js/package`
+3. `yarn`
+4. `yarn build` for use in browser or `yarn build-node` for use in node.js
+5. Add `"manta.js": "file:/<LOCAL PATH OF SDK/manta-js/package>` to your project's package.json
+6. `yarn upgrade manta.js` in your project's directory
+
+# Usage
+
+All methods are called through the `MantaPrivateWallet` class.
+
+[manta-signer](#signer) must be installed and running.
```

Comment:
Maybe we should tell readers where to install signer.

Reply:
Yes this links to the section at the bottom of the page.
I can add more explanation there, let me know.